dds_web/commands.py

```diff
@@ -225,21 +228,26 @@ def update_unit_quota(unit_id, quota):
         flask.current_app.logger.error(f"There is no unit with the public ID '{unit_id}'.")
         sys.exit(1)
 
+    # calculate quotas for logging purposes
+    old_quota_tb = round(unit.quota / 1000**4, 4)
+    new_quota_bytes = int(quota * 1000**4)
+
     # ask the user for confirmation
     do_update = rich.prompt.Confirm.ask(
-        f"Current quota for unit '{unit_id}' is {round(unit.quota / 1000 ** 3,2)} GB. \n"
-        f"You are about to update the quota to {quota} GB ({quota * 1000 ** 3} bytes). \n"
+        f"Current quota for unit '{unit_id}' is {old_quota_tb} TB. \n"
+        f"You are about to update the quota to {quota} TB. \n"
+        f"NOTE: In the database it will appear as bytes: {new_quota_bytes} Bytes \n"
         "Are you sure you want to continue?"
     )
     if not do_update:
         flask.current_app.logger.info(
-            f"Cancelling quota update for unit '{unit_id}'. The quota is still {round(unit.quota / 1000 ** 3,2)} GB. ({unit.quota} bytes.)"
+            f"Cancelling quota update for unit '{unit_id}'. The quota is still {old_quota_tb} TB."
         )
         return
 
-    # Set sto4 info
-    quota_bytes = quota * 1000**3
-    unit.quota = quota_bytes
+    # Set new quota info
+    new_quota_bytes = int(quota * 1000**4)
+    unit.quota = new_quota_bytes
     db.session.commit()
 
     flask.current_app.logger.info(f"Unit '{unit_id}' updated successfully")
```

Comment:
Probably good opportunity to add a try-except block for error handling, something like:

```suggestion
try:
    unit.quota = new_quota_bytes
    db.session.commit()
    flask.current_app.logger.info(f"Unit '{unit_id}' updated successfully")
except Suitable_exception as e:
    db.session.rollback()
    flask.current_app.logger.error(f"Failed to update quota for unit '{unit_id}': {str(e)}")
    sys.exit(1)
```